tests/attestation/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+volumes:
+  uds:
+services:
+  primary:
+    image: $REGISTRY/attestation/primary:$TAG
+    build:
+      context: ../../
+      dockerfile: tests/attestation/primary.Dockerfile
+    network_mode: "host"
+    ports:
+      - "8000:8000"
+  attestation:
+    image: $REGISTRY/attestation/sidecar:$TAG
+    build:
+      context: ../../
+      dockerfile: docker/attestation-container/Dockerfile.run
+    command: ["app", "-socket-address", "/mnt/uds/sock"]
+    network_mode: "host"
+    ports:
+      - "8080:8080"
+    environment:
+      - AZURE_ATTESTATION_ENDPOINT=cacisidecars.eus.attest.azure.net
```

Comment:
Do we want this as a secret/env variable as well?

Reply:
https://github.com/microsoft/confidential-sidecar-containers/pull/115/commits/05922b3fb3d0743ec4af3730a125f5e9f68461e4